test/package_oss_test.go

```diff
@@ -165,7 +165,7 @@ echo $token
 				_, stderr, err := RunCommand(PackageName, variableIdentifier)
 
 				assert.Error(t, err)
-				assert.Contains(t, stderr.String(), "not found in account")
+				assert.Contains(t, stderr.String(), "CONJ00076E Variable cucumber:variable:non-existent-variable is empty or not found")
```

Comment:
Just a fly by comment. One of the reasons I like error contains is that it simplifies asserting on errors